frontend/src/v4/modules/viewerGui/viewerGui.sagas.ts

```diff
@@ -386,7 +386,7 @@ function* loadModel() {
 		const content = 'The model was either not found, failed to load correctly ' +
 			'or you are not authorized to view it. ' +
 			' You will now be redirected to the teamspace page.';
-		yield put(DialogActions.showDialog({ title: 'Model Error', content }));
+		yield put(DialogActions.showTextOnlyDialog({ title: 'Model Error', content }));
```

Comment:
Should this be a "Redirect to Teamspace" dialog?

Reply:
If you are talking about what the sagas file is concerned about, maybe. What I have done here was simply restyling the component. A change in functionality should be addressed in another issue I guess